assets/client/src/components/ChallengeDetails.js

```diff
@@ -404,10 +404,10 @@ export const ChallengeDetails = ({challenge, challengePhases, preview, print, ta
                 </div>
               }
               {!print &&
-                <a className="follow__btn" href={`${apiUrl}/public/previews/challenges?challenge=${encodeURIComponent(challenge.uuid)}&print=true`} target="_blank" rel="noopener noreferrer">
+                <a className="follow__btn" href={`${apiUrl}/public/previews/challenges/${challenge.custom_url || challenge.id}?print=true`} target="_blank" rel="noopener noreferrer">
```

Comment:
The `ChallengeDetails` view component isn't really used anymore, since it's been replace with the Rails view, but this route is invalid and should be reverted.
```suggestion
                <a className="follow__btn" href={`${apiUrl}/public/previews/challenges?challenge=${encodeURIComponent(challenge.uuid)}&print=true`} target="_blank" rel="noopener noreferrer">
```